opendevin/runtime/docker/ssh_box.py

```diff
@@ -441,11 +445,38 @@ def execute(
         self.ssh.sendline(cmd)
         if stream:
             return 0, SSHExecCancellableStream(self.ssh, cmd, self.timeout)
-        success = self.ssh.prompt(timeout=timeout)
-        if not success:
-            logger.exception('Command timed out, killing process...', exc_info=False)
-            return self._send_interrupt(cmd)
-        command_output = self.ssh.before
+        prompts = [
```

Comment:
WIll we move to Paramiko for windows support?

Reply:
In the next week, I'm thinking about completely get rid of SSH, and spin up a websocket client inside sandbox, connect it to the backend, and do things from there -- this will allow us to use ONE event stream for everything which can be cleaner. I can start a draft PR and we can discuss from there i guess.

Reply:
Yeah, that would be better for connecting the frontend terminal.

Reply:
> How about we inherit from `pxssh.pxssh`, and override their `.prompt` method to implement this? It can make the code much more readable?

In which file, do you want this to be implemented?

Reply:
still need to inherit as it reaches the end of life?

Reply:
I don't think so! We can leave it as is for now probably?

Reply:
Yes.

Reply:
Could you reopen it?